src/scss/grommet-core/_objects.search.scss

```diff
@@ -138,17 +138,90 @@
   }
 
   &--large {
-    line-height: triple($inuit-base-spacing-unit);
-
     .search__input {
-      @include inuit-font-size($large-number-font-size, double($inuit-base-spacing-unit));
-      padding: halve($inuit-base-spacing-unit - $large-number-font-size - $input-border-width)
-        ($large-number-font-size - $input-border-width);
-      padding-right: (double($inuit-base-spacing-unit) - $input-border-width);
+      @include inuit-font-size($large-input-font-size, double($inuit-base-spacing-unit));
+      padding: halve($inuit-base-spacing-unit) ($inuit-base-spacing-unit);
+      padding-right: ($inuit-base-spacing-unit * 3);
+      line-height: normal;
```

Comment:
This is overriding what Inuit is doing in inuit-font-size at line 142. Why do we need the line-height here?